security/cmd/istio_ca/main.go

```diff
@@ -408,7 +408,7 @@ func runCA() {
 		}
 	}
 
-	monitorErrCh := make(chan error)
+	monitorErrCh := make(chan error, 10)
```

Comment:
Any reason to set the buffer size 10?

Reply:
honestly, I just picked a random number to test it out and forgot to think about it more. But I think any number above 1 will do, although I didn't look too closely to make sure we send exactly 1 error